connaisseur/flask_server.py

```diff
@@ -111,10 +112,4 @@ def readyz():
     except HTTPError:
         webhook_response = None
 
-    notary_health = health_check(os.environ.get("NOTARY_SERVER"))
-
-    return (
-        ("", 200)
-        if ((webhook_response or sentinel_running) and notary_health)
-        else ("", 500)
-    )
+    return ("", 200) if (webhook_response or sentinel_running) else ("", 500)
```

Comment:
Since this changes behaviour from ADR-1, we might want to add an update as ADR-3. Would probably be a good idea anyways, covering the config layout we discussed at length